frontend/src/main/pages/AdminLoadSubjectsPage.js

```diff
@@ -1,14 +1,61 @@
 import BasicLayout from "main/layouts/BasicLayout/BasicLayout";
+import { useBackendMutation, useBackend } from "main/utils/useBackend";
+import { toast } from "react-toastify";
+import { Button } from "react-bootstrap";
+import UCSBSubjectsTable from 'main/components/UCSBSubjects/UCSBSubjectsTable';
+import { useCurrentUser } from 'main/utils/currentUser'
+
+
 
 export default function AdminLoadSubjectsPage() {
+
+  const currentUser = useCurrentUser();
+  const { data: subjects, error: _error, status: _status } =
+      useBackend(
+        // Stryker disable next-line all : don't test internal caching of React Query
+        ["/api/UCSBSubjects/all"], { method: "GET", url: "/api/UCSBSubjects/all" }, []
+      );
+
+  const objectToAxiosParams = (subjects) => ({
+    url: '/api/UCSBSubjects/load',
+    method: 'POST',
+  });
+
+  var subjectsCount = subjects.length;
+
+  const onSuccess = (subjects) => {
+    toast(`Number of Subjects Loaded : ${subjects.length - subjectsCount}`);
```

Comment:
The use of the new var instead of trying to somehow retain the old subjectsCount. This is what I was looking for. Well done.